### DISCUSSION TITLE
адрес получения  платежа

### DISCUSSION THREAD

**Discussion by mikchailti** [Chia-Network/chia-blockchain#8175] [General]
Адрес для получения платежа отличается от адреса на фарминге, это нормально ?! по сути награда уйдет кому то другому ?

**Comment by BrandtH22** [comment]
@mikchailti , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - each keyset in chia has a nearly infinite number of associated xch addresses, to verify the xch address used for farming is controlled by the keyset on the machine you can follow the below instructions while using that xch address on the address tab rather than an NFT id on the NFT tab: https://docs.chia.net/faq/#how-can-i-sign-and-verify-a-message-with-an-nft

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
